runtime_data/postprocess/gb.patterns

```diff
@@ -1 +1,33 @@
-gb	@@##@@@
+gb	[A-HK-PR-TV-Y][A-HJ-PRSTV-Y]##[A-HJ-PR-Z][A-HJ-PR-Z][A-HJ-PR-Z]
+
+gb	[A-HJ-NP-TV-Y]#[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+gb	[A-HJ-NP-TV-Y]##[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+gb	[A-HJ-NP-TV-Y]###[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]
+
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+gb	[A-HJ-PRSTV-Y][A-HJ-PRSTV-Y][A-HJ-PRSTV-Y]#[A-HJ-NP-TV-Y]
+
+gb	###[A-HJ-PR-Y][A-HJ-PR-Y][A-HJ-PR-Y]
```

Comment:
Allow for Northern Irish plates (with an I or Z) with a leading number
eg.
`1 IA` was issued in Co Antrim in 1958, but could still be used as a personal plate
Lord Mayor of Belfast car has the old Belfast plate `1 WZ`

(Also in the near future, Fermanagh plates could end up with leading 4 numbers and 3 letters including an 'I' or 'Z')

Reply:
Thanks for the feedback! Resolved in commit 82cda89ffd0564a6292b423c9837f9cbf5db818b